andela/hc-leo
# Ft configure nagging interval #157694798

**What does this PR do?**
This PR adds the nag feature. 

**Description of Task to be completed?**
Initially, when a User's job goes down, the user is sent a notification email informing him of the same. Currently, the system keeps nagging the user every so often until they resolve the job that has gone down.

**How should this be manually tested?**
- Clone the repo, ```cd``` into the project and run ```./manage.py runserver```. 
- Navigate to the URL provided by the development server (usually ```localhost```) using a browser.
- Create an account or log in
- Create a new check and with a suitable ```period```, ```grace``` and ```nag```
- Ping the URL provided.
- Wait until the job goes down. When the job is not resolved, the job color changes to purple and the system keeps on nagging you till the job is resolved.


**What are relevant pivotal tracker stories?**
[#157694798](https://www.pivotaltracker.com/story/show/157694798)
